ingestion/src/metadata/orm_profiler/api/workflow.py

```diff
@@ -257,7 +269,7 @@ def create_profiler_interface(
                 ometa_client=create_ometa_client(self.metadata_config),
                 thread_count=self.source_config.threadCount,
                 table_entity=self._table_entity,
-                table_sample_precentage=self.get_profile_sample(self._table_entity)
+                table_sample_profile=self.get_profile_sample(self._table_entity)
```

Comment:
I think we should make `profile_sample` and `profile_sample_type` attributes of the object (vs passing them as dictionary). Right now it forces us to do something a bit convoluted in `core.py` ln 495-516.

We can have `profile_sample_type` default to `ProfileSampleType.PERCENTAGE`